### ISSUE TITLE
The time cost is far longer than '20ms' with gpu.

### ISSUE THREAD

**Issue by xscjun** [youngwanLEE/CenterMask#2]
Hi，Thanks for your open source code. 
 I run the centermask_demo.py with the model "centermask-lite-M-v2-ms-bs32-1x.pth",and config file "centermask_M_v2_FPN_lite_res600_ms_bs32_1x.yaml",
 the time cost is 60ms-110ms since the second image :

file 0
COCO_val2014_000000128654 processing...
COCO_val2014_000000128654       inference time: 0.281s
file 1
COCO_val2014_000000007281 processing...
COCO_val2014_000000007281       inference time: 0.072s
file 2
COCO_val2014_000000005477 processing...
COCO_val2014_000000005477       inference time: 0.068s
file 3
COCO_val2014_000000463842 processing...
COCO_val2014_000000463842       inference time: 0.111s
file 4
COCO_val2014_000000053505 processing...
COCO_val2014_000000053505       inference time: 0.075s
file 5
COCO_val2014_000000000885 processing...
COCO_val2014_000000000885       inference time: 0.068s
file 6
COCO_val2014_000000033759 processing...
COCO_val2014_000000033759       inference time: 0.067s
file 7
COCO_val2014_000000050896 processing...
COCO_val2014_000000050896       inference time: 0.119s
file 8
COCO_val2014_000000012639 processing...
COCO_val2014_000000012639       inference time: 0.075s
file 9
COCO_val2014_000000479030 processing...
COCO_val2014_000000479030       inference time: 0.070s
file 10
COCO_val2014_000000001000 processing...
COCO_val2014_000000001000       inference time: 0.076s
file 11
COCO_val2014_000000039769 processing...
COCO_val2014_000000039769       inference time: 0.120s

The  test image size set :
INPUT:
  MIN_SIZE_RANGE_TRAIN: (580, 600)
  MAX_SIZE_TRAIN: 1000
  MIN_SIZE_TEST: 600
  MAX_SIZE_TEST: 1000

Even I set the MAX_SIZE_TEST to 500, MIN_SIZE_TEST to 300,the time cost is still far longer than “20”ms as you mentioned.   

My pytorch version : 1.3.1
CUDA:10.0.1
GPU: P40

How can I improve time efficiency，please？

**Comment by youngwanLEE** [comment]
@xscjun ,

centermask_demo.py code is not optimized yet.

the inference time you measured includes all visualization steps that are time-consuming.

In general,

Many papers report their inference time using only model inference time except for visualization.

In addition, 

Using below command, you can get about 20ms model inference time on coco validation set.

`python tools/test_net.py --config-file "configs/centermask/centermask_M_v2_FPN_lite_res600_ms_bs32_1x.yaml" MODEL.WEIGHT centermask-lite-M-v2-ms-bs32-1x.pth TEST.IMS_PER_BATCH 1`

At last, 

we'll further optimize the demo program like [YOLACT code ](https://github.com/dbolya/yolact/blob/master/eval.py).

**Comment by xscjun** [comment]
> @xscjun ,
> 
> Hi, thanks for your attention.
> 
> centermask_demo.py code is not optimized yet.
> 
> The inference time you measured includes all visualization steps that are time-consuming.
> 
> In general,
> 
> Many papers report their inference time using only model inference time except for visualization.
> 
> If you remove or turn-off the visualization step, you can get further faster speed.
> 
> In addition,
> 
> Using below command, you can get about 20ms model inference time on coco validation set.
> 
> `python tools/test_net.py --config-file "configs/centermask/centermask_M_v2_FPN_lite_res600_ms_bs32_1x.yaml" MODEL.WEIGHT centermask-lite-M-v2-ms-bs32-1x.pth TEST.IMS_PER_BATCH 1`
> 
> At last,
> we'll further optimize the demo program like [YOLACT code ](https://github.com/dbolya/yolact/blob/master/eval.py).
> 
> When we finish the demo code, we'll notice you again.

OK , thanks.

**Comment by zimenglan-sysu-512** [comment]
any update?
